internal/fs/inode/file_test.go

```diff
@@ -928,3 +928,22 @@ func (t *FileTest) UnlinkLocalFile() {
 	AssertNe(nil, err)
 	AssertEq("gcs.NotFoundError: Object test not found", err.Error())
 }
+
+func (t *FileTest) TestMultipleCallsToWriteToBufferCreatesBufferOnce() {
```

Comment:
unit tests for ensureBuffer to validate that inmemory buffer is created for <50MB